src/main/java/com/alipay/oceanbase/rpc/stream/ObTableClientQueryStreamResult.java

```diff
@@ -125,21 +139,33 @@ protected ObTableQueryResult execute(ObPair<Long, ObTableParam> partIdWithIndex,
                                     indexTableName, partIdWithIndex.getLeft(), e.getMessage(), e);
                             throw e;
                         }
-                    } else if (e instanceof ObTableException
-                               && ((ObTableException) e).isNeedRefreshTableEntry()) {
-                        needRefreshTableEntry = true;
-                        logger
-                            .warn(
-                                "tablename:{} partition id:{} stream query refresh table while meet Exception needing refresh, errorCode: {}",
-                                indexTableName, partIdWithIndex.getLeft(),
-                                ((ObTableException) e).getErrorCode(), e);
-                        if (client.isRetryOnChangeMasterTimes()
-                            && (tryTimes - 1) < client.getRuntimeRetryTimes()) {
+                    } else if (e instanceof ObTableException) {
```

Comment:
OB_TABLE_NOT_EXIST is an ObTableEntryRefreshException. You could use the previous code to enter this block. And deal with table group inside the previous block

Reply:
edit , clear table group cache